plugins/filter_aws/aws.h

```diff
@@ -106,11 +85,29 @@ struct flb_filter_aws {
     size_t account_id_len;
     int account_id_include;
 
-
     flb_sds_t hostname;
     size_t hostname_len;
     int hostname_include;
 
+    /* tags_* fields are related to exposing EC2 tags in log labels
+     * tags_include defines if EC2 tags functionality is enabled */
+    int tags_include;
+    /* tags_fetched defines if tag keys and values were fetched successfully
+     * and might be used to inject into msgpack */
+    bool tags_fetched;
```

Comment:
In Fluent Bit, we use `int` for boolean and use the values `FLB_TRUE` and `FLB_FALSE`.